src/app/devices/deviceIdentity/components/deviceIdentity.tsx

```diff
@@ -104,10 +105,18 @@ export const DeviceIdentityInformation: React.FC<DeviceIdentityDataProps & Devic
 
     const renderSasTokenSection = () => {
         return (
-            <SasTokenGenerationView
-                activeAzureResourceHostName={hostName}
-                deviceIdentity={state.identity}
-            />
+            <>
+                <SasTokenGenerationView
+                    activeAzureResourceHostName={hostName}
+                    deviceIdentity={state.identity}
+                />
+                <br />
+                <QrGenerationView
```

Comment:
i am wondering if it should be a separate nav item. And are we going to update the readme or just keep it low profile?